src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -211,19 +212,28 @@ const stocktakesInitialiser = () => {
  * @param    {StocktakeItem}  stocktakeItem
  * @returns  {object}
  */
-const stocktakeBatchInitialiser = stocktakeItem => ({
-  pageObject: stocktakeItem,
-  backingData: stocktakeItem.batches,
-  data: stocktakeItem.batches.slice(),
-  keyExtractor: recordKeyExtractor,
-  dataState: new Map(),
-  sortKey: 'itemName',
-  isAscending: true,
-  modalKey: '',
-  modalValue: null,
-  columns: getColumns(ROUTES.CUSTOMER_INVOICE),
-  getPageInfoColumns: getPageInfoColumns(ROUTES.CUSTOMER_INVOICE),
-});
+const stocktakeBatchInitialiser = stocktakeItem => {
+  const thisStoreNameId = UIDatabase.getSetting(SETTINGS_KEYS.THIS_STORE_NAME_ID);
+  const suppliers = UIDatabase.objects('Name').filtered(
+    '(isVisible == true AND isSupplier == true) AND id != $0 OR (type == "inventory_adjustment")',
```

Comment:
I'm not sure about the intention of that first set of brackets:

```suggestion
    '(isVisible == true AND isSupplier == true AND id != $0) OR (type == "inventory_adjustment")',
```